org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLGeneralClientSettings.java

```diff
@@ -17,7 +17,7 @@
 /**
  * Class to hold all settings from the client side.
  * 
- * See https://github.com/eclipse/lemming/wiki/Configuration for more
+ * See https://github.com/rzgry/lemminx/blob/lemminx-ext-docs/docs/Configuration.md for more

```

Comment:
Change to `https://github.com/eclipse/lemminx/blob/master/docs/Configuration.md`

Reply:
Whoops 😄 good catch fixed.